<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> This file is part of TUNNEX.

 TUNNEX is free software: you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation, either version 3 of the License, or
 laster any later version.

 TUNNEX is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.

 You should have received a copy of the GNU General Public License
 along with TUNNEX.  If not, see http://www.gnu.org/licenses/.

 TUNNEX 1.0.0 
 Copyright (c) Henrik Quanz
 </comment>
    <class>WKB::widgets::PlotControl</class>
 <widget class="QWidget" name="PlotControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="xMaxEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="FloatingPointQSlider" name="xMaxSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="FloatingPointQSlider" name="xMinSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="FloatingPointQSlider" name="yMaxSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="FloatingPointQSlider" name="yMinSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="xMinEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="yMinEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="yMaxEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatingPointQSlider</class>
   <extends>QSlider</extends>
   <header>controller/floatingPointQSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
